elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java

```diff
@@ -697,6 +698,30 @@ public interface ElytronSubsystemMessages extends BasicLogger {
     @Message(id = 1211, value = "Unable to load the credential store.")
     OperationFailedException unableToLoadCredentialStore(@Cause Throwable cause);
 
+    @Message(id = 1212, value = "KeyStore does not contain a PrivateKey for KeyStore: [%s] and alias: [%s].")
+    StartException invalidPrivateKey(String keyStore, String alias);
```

Comment:
@Ashpan Just a minor, the names of these messages could be updated to missingPrivateKey and missingPublicKey to be more precise

Reply:
Updated